system/datatypes/mailinglist.php

```diff
@@ -1010,6 +1062,8 @@ protected function mailingSendAction(HyphaRequest $request) {
 		$this->xml->lockAndReload();
 		$mailing = $this->xml->getElementById($mailingId);
 		$mailing->setAttribute('status', self::MAILING_STATUS_SENDING);
+		$count = 0;
+		$mailing->setAttribute('progress', $count);
```

Comment:
This should use a constant. Same for `status` btw, and some other string constants in this function, but those would be a separate comit (would be good if you could fix them up, though).

Reply:
Done :)